graphene_django_cud/mutations/core.py

```diff
@@ -165,13 +163,13 @@ def get_or_upsert_m2o_objs(cls, obj, field, values, data, operation, info, Model
                 }
 
                 if field_type == "auto":
-                    # In this case, a new type has been created for us. Let's first find it's name,
-                    # then get it's meta, and then create it. We also need to attach the obj as the
+                    # In this case, a new type has been created for us. Let's first find its name,
+                    # then get its meta, and then create it. We also need to attach the obj as the
                     # foreign key.
-                    _type_name = data.get(
-                        "type_name",
-                        f"{operation.capitalize()}{Model.__name__}{field.name.capitalize()}",
-                    )
+                    # _type_name = data.get(
+                    #     "type_name",
+                    #     f"{operation.capitalize()}{Model.__name__}{field.name.capitalize()}",
+                    # )
```

Comment:
Why this?

Reply:
Don't know why I commented this out instead of removing it. Removed now.